ls1intum/Artemis
# [Bugfix] Fix min-width of plant UML

### Checklist

### Motivation and Context


The students should have a clear view of the UML diagram in the problem statement. Especially when using the code editor which only offers a small area for the problem statement

### Description


### Steps for Testing

This PR fixes the problem that the UML diagrams in problem statements get squashed, when the vertical area of the problem statement is not big enough. Instead a scrollbar appears.

1. Log in to Artemis
2. Navigate to an exercise with a wide UML diagram (For example the standard Java exercise with the strategy pattern)
3. Check that the UML diagram is not squashed regardless of the window width
4. Check that the diagram is also displayed correctly in the code editor on the right hand side

### Screenshots



An old squashed diagram:
![Bildschirmfoto 2021-07-10 um 12 25 41](https://user-images.githubusercontent.com/38322605/125159802-015c0680-e17a-11eb-8571-8f9965bc8985.png)

A new non squashed diagram with a scrollbar at the bottom:
![Bildschirmfoto 2021-07-10 um 12 25 54](https://user-images.githubusercontent.com/38322605/125159811-19cc2100-e17a-11eb-87b7-232139aa475c.png)
